dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/route/JaxRsRouteTestServlet.java

```diff
@@ -111,6 +111,70 @@ public void testRouteWithQueryParam() {
                         .withAttribute(SemanticAttributes.HTTP_ROUTE, getPath() + "/getWithQueryParam"));
     }
 
+    @Test
+    public void testRouteWithSubResourceWithPathParam() {
+        URI testUri = getUri();
+        Span span = utils.withTestSpan(() -> {
+            Response response = ClientBuilder.newClient().target(testUri).path("getSubResourceWithPathParam/myIdForTesting/details").request()
+                            .build("GET").invoke();
+            System.out.println("RESPONSESSS" + response.toString());
+            assertThat(response.getStatus(), equalTo(200));
+            assertThat(response.readEntity(String.class), equalTo("myIdForTesting"));
+        });
+
+        List<SpanData> spans = exporter.getFinishedSpanItems(3, span.getSpanContext().getTraceId());
+        TestSpans.assertLinearParentage(spans);
+
+        SpanData clientSpan = spans.get(1);
+        SpanData serverSpan = spans.get(2);
+
+        assertThat(clientSpan, isSpan()
+                        .withKind(SpanKind.CLIENT)
+                        .withAttribute(SemanticAttributes.HTTP_METHOD, "GET")
+                        .withAttribute(SemanticAttributes.HTTP_STATUS_CODE, 200L)
+                        .withAttribute(SemanticAttributes.HTTP_URL, testUri.toString() + "getSubResourceWithPathParam/myIdForTesting/details"));
+
+        assertThat(serverSpan, isSpan()
+                        .withKind(SpanKind.SERVER)
+                        .withAttribute(SemanticAttributes.HTTP_METHOD, "GET")
+                        .withAttribute(SemanticAttributes.HTTP_STATUS_CODE, 200L)
+                        .withAttribute(SemanticAttributes.HTTP_ROUTE, getPath() + "getSubResourceWithPathParam/{id}/details")
+                        .withAttribute(SemanticAttributes.HTTP_TARGET, getPath() + "getSubResourceWithPathParam/{id}/details"));
```

Comment:
```suggestion
                        .withAttribute(SemanticAttributes.HTTP_TARGET, getPath() + "getSubResourceWithPathParam/myIdForTesting/details"));
```

Reply:
Done

Reply:
Looks like this change has got lost somewhere